### ISSUE TITLE
GEDCOM data after sign-in - 2.0.0

### ISSUE THREAD

**Issue by melizaa** [fisharebest/webtrees#2429]
When I sign-in to the test systems, I expect to see as the defauld the data of the default GEDCOM.

Now I see the data of other GEDCOMS. It seems that they are the GEDCOMS whose name is sorted as the 1st one.

**Comment by fisharebest** [comment]
This works for me.

Can you give the exact sequence of events to reproduce this.

When you click the link to the login page, the current page is in included in the URL.
After login, you are redirected back to this page.

If there was no previous page (because you arrived directly at the login page), **and** your login account is linked to an individual, then you will be redirected to that tree.

**Comment by melizaa** [comment]
When I am signed out, I see the default GEDCOM.
The top sign-in works as expected.

The block sign-in transfers me to the 1st by name sorted GEDCOM.

**Comment by melizaa** [comment]
Please add the sign-in block to the home-page of demo-dev, so I can see if the issue exist also there.

**Comment by xmlf** [comment]
If different family trees can bind different domain names, it will be very good!

**Comment by melizaa** [comment]
Logging in using the Sign in block does not yet seem to work.

**Comment by fisharebest** [comment]
> Logging in using the Sign in block does not yet seem to work.

It works for me.

Can you explain what isn't working for you.

**Comment by melizaa** [comment]
I downloaded the changed file and transferred it yesterday to the test and
future test systems.
I enter one of the two test systems in Google Chrome and see the default
GEDCOM data.
I sign in as administrator using the block. I see the 1st (sorted by name)
GEDCOM's data.

I cleaned cookies and cached images and files.

After I sign-in I see still as the default the data of the wrong GEDCOM.

I test now in Microsoft Edge and FireFox. Still the same behavior.

I sign-in as pgvdev in FF. Same issue.

Meliza
